manifests/s/SlackedLime/TimeSpent/1.2/SlackedLime.TimeSpent.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: SlackedLime.TimeSpent
+PackageVersion: 1.2
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/slackedlime/TimeSpent/releases/download/1.2/TimeSpent.Setup.v1.2.exe
+  InstallerSha256: 37157AC99D7B9FF4E8B8B2365648817478D6425D54200EA9491151F987542720

```

```suggestion
  InstallerSha256: 37157AC99D7B9FF4E8B8B2365648817478D6425D54200EA9491151F987542720
  AppsAndFeaturesEntries:
  - DisplayName: v1.2
```
AppsAndFeaturesEntries are required because the Display name is not similar to the Package Name